src/Setup.php

```diff
@@ -20,7 +20,8 @@ public function __construct(){
 	  if( apply_filters('clarkon_event_load_default_acf_fields', true) ) {
 		$this->loadDefaultACFFields();
 	  }
-	  
+
+	  load_plugin_textdomain( 'clarkson-event-manager', false, dirname( plugin_basename( __FILE__ ) ) . '/languages' );
```

Comment:
```suggestion
```

Deze hoeft niet opnieuw geladen te worden, gezien je dat al in `clarkson_event_manager_init_plugin()` doet, toch?